Description: Transtion to Gradle 3.1 
 This patch makes the build scripts compatible with Gradle 3.1 
Author: Kai-Chung Yan (殷啟聰) 
Last-Update: 2016-11-02
--- a/build.gradle
+++ b/build.gradle
@@ -293,7 +293,11 @@
     header = packageTitle
     footer = copyrightString
     include 'groovyx/gpars/**'
-    overview = new File('overview.html')
+    if (gradle.gradleVersion.startsWith('2.')) {
+        overview = new File('overview.html')
+    } else {
+        overviewText = resources.text.fromFile('overview.html')
+    }
     groovyClasspath = files('/usr/share/java/groovy-all.jar')
 }
 
